bin/extract.sh

```diff
@@ -20,3 +30,5 @@ do
     tar -xzf $corpus "$name"/"$name"-sentences.txt -O \
         | awk '!($1="")' > txt/"$name".txt
 done
+
+
```

Comment:
Nitpick, only one trailing line please. :-)

Note that Vim always leaves an empty line (which is a good practice) but doesn’t display it (which can be confusing). So if you’re using Vim, there should be no visible trailing lines in your window.